viewer/packages/api/visual-tests/Image360API.VisualTest.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+import * as THREE from 'three';
+import { degToRad } from 'three/src/math/MathUtils';
+import dat from 'dat.gui';
+
+import {
+  ViewerTestFixtureComponents,
+  ViewerVisualTestFixture
+} from '../../../visual-tests/test-fixtures/ViewerVisualTestFixture';
+
+// Sanity test for loading model
+export default class DefaultVisualTest extends ViewerVisualTestFixture {
+  public async setup({ viewer }: ViewerTestFixtureComponents): Promise<void> {
```

Comment:
I suppose this could be renamed for clean-ness. (Also, doesn't work, but at least we've agreed that it doesn't.)